flutter/shell/platform/tizen/tizen_window_elementary.cc

```diff
@@ -403,4 +444,74 @@ void TizenWindowElementary::PrepareInputMethod() {
       [this](std::string str) { view_delegate_->OnCommit(str); });
 }
 
+#ifdef EOM_SUPPORT
```

Comment:
Apparently the EOM module itself is available on all profiles. 

https://docs.tizen.org/application/native/api/wearable/4.0/group__CAPI__UI__EOM__MODULE.html
https://docs.tizen.org/application/native/api/mobile/5.5/group__CAPI__UI__EOM__MODULE.html

c.f. This is not the case for `tzsh_softkey` which is not available on wearable 4.0.

How about removing the `#ifdef` guards and making the code common to all profiles?